instrumentation/pg/test/opentelemetry/instrumentation/pg/patches/connection_test.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+require 'active_record'
+require 'pg'
+
+require_relative '../../../../../lib/opentelemetry/instrumentation/pg'
+require_relative '../../../../../lib/opentelemetry/instrumentation/pg/patches/connection'
```

Comment:
Are these requires necessary? i.e. is the `test_helper` and bundler not requiring these already?

Reply:
NVM Looks like they are... 😕